lib/steps.js

```diff
@@ -177,6 +177,7 @@ When(/^the user gets to ?(?:the)?(?: ?question)?(?: id)? "([^"]+)" with this dat
   // still have to be some automatic continuing.
   // It also seems harsh, but more reliable.
   // Ask developers what they'd like.
+  let count = 0;
```

Comment:
My comments on the other PR #168 carry over here too, since this one builds on that one.

Reply:
Made #171